pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/data/samples/SamplesBBC.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.demo.shared.data.samples
+
+import ch.srgssr.pillarbox.demo.shared.data.DemoItem.URL
+import ch.srgssr.pillarbox.demo.shared.data.Playlist
+
+/**
+ * Samples from BBC
+ */
+@Suppress("StringLiteralDuplication", "MaximumLineLength", "MaxLineLength", "UndocumentedPublicProperty")
+object SamplesBBC {
+
+    val HlsTestCardAVC = URL(
+        title = "[HLS ]Live HLS Audio and AVC video",
```

Comment:
```suggestion
        title = "[HLS] Live HLS Audio and AVC video",
```

Reply:
@MGaetan89  We are looking for people able to distinguish left spaces from right spaces. Are you available to discuss this position?

Reply:
I might be overqualified for that. I am also able to distinguish two spaces from one (https://github.com/SRGSSR/pillarbox-android/pull/1176#discussion_r2378007359) 🤣